.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Description
+Please include a summary of the change and which issue is fixed. Include the motivation for the changes, and comment on your PR if necessary for clarity.
+
+Fixes: (link Asana Tank)
+
+# Type of change
+
+- [ ] Bug fix (change which fixes an issue)
+- [ ] New feature (change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+Please validate that you have tested your change in at least one of the following areas:
+
+- [ ] Successfully tested locally (or docker image)
+- [ ] Successfully tested on a staging or sandbox server
```

Comment:
My only comment here is what officially qualifies as a staging or sandbox server?

Reply:
It's a little murky honestly. I think by process of elimination on my other two bullets, it's "a hosted server that's not my local development environment that doesn't belong to a merchant"